rules/windows/credential_access_disable_kerberos_preauth.toml

```diff
@@ -0,0 +1,68 @@
+[metadata]
+creation_date = "2022/01/24"
+maturity = "production"
+updated_date = "2022/01/24"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the modification of an account Kerberos preauthentication options. An adversary with GenericWrite/GenericAll rights over
+the account can maliciously modify these settings to perform offline password cracking attacks such as AS-REP roasting.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-windows.*", "logs-system.*"]
```

Comment:
will this work with the windows integration?

Reply:
I'm maintaining it on this kind of rules for "Forwarded Events", but it would require system integration to collect the security logs I think.